### PR TITLE
Eren editing home screen

### PR DESCRIPTION
**Pull request by Enmoren**
Relevant trello cards:
- https://trello.com/c/5S8riEo1/130-react-redesign-the-starter-page
- https://trello.com/c/kKdHKelu/133-react-edit-home-screen-buttons
- https://trello.com/c/JPo1AlFs/128-react-put-the-delete-vehicle-section-button-in-the-vehicle-section-header

### CODE REVIEW

File: components/home/Welcome.js [hunk 1]
```diff
@@ -18,10 +18,13 @@ class Welcome extends Component {
...
+                    <View style={styles.btnContainer}>
+                        <TouchableOpacity style={styles.styledButton} activeOpacity = { .7 } onPress={() => navigateTo('Survey')}>
+                            <Text style={styles.btnText}>Quick Start</Text>
+                        </TouchableOpacity>
+                    </View>
+               </Layout>
               <Layout style={styles.bottomBar}>
-                <TouchableOpacity onPress={()=>Linking.openURL('https://forms.gle/ho3cZNyoaFArNNN79')}><Text style={{color: 'blue'}}>Submit Feedback</Text></TouchableOpacity>
                 <Text style={styles.bottomBarText}>
                   {"Built by students at Olin College of Engineering 2020"}
```
Comment (coverney) [inline components/home/Welcome.js:29]: Should be "Built by students at Olin College of Engineering 2020-2021"?

File: components/home/homeSections/VehicleSection.js [hunk 2]
```diff
@@ -62,16 +62,22 @@ class VehicleSection extends Component{
...
-        //       <Icon name='minus-circle' width={25} height={25} />
-        //     </View>
-        // );
+//         would need to refine if we want to put delete vehicle section icon next to card header
+         const VehiclesEditHeader = () => (
+            <CardHeader>
+             <View >
+               <Text style={styles.cardHeadertxt}>{`${name} ${index + 1}`}</Text>
+             </View>
+             <Icon name='minus-circle' width={25} height={25} onPress={() => this.setState({beforeVehicleDelete:true})}/>
```
Comment (coverney) [inline components/home/homeSections/VehicleSection.js:78]: can we add more spacing between Vehicle text and the minus button? It looks a bit tight.

File: components/home/Home.js [hunk 3]
```diff
@@ -139,7 +139,7 @@ class Home extends Component {
                                 this.setState({edit: true})
                            }}
                            />
-                           <Text style={styles.rightControlsText}>Edit</Text>
+                           <Text style={styles.rightControlsText}>Edit Sections</Text>
```
Comment (coverney) [inline components/home/Home.js:142]: Should we have a newline between "Edit" and "Sections"?

File: components/home/Home.js [hunk 4]
```diff
@@ -107,20 +107,20 @@ class Home extends Component {
...
+//            const { edit } = this.state
             if (edit) {
                 return (
                     <View style={styles.rightControlsContainer}>
                         <Layout style={styles.rightControls}>
                             <TopNavigationAction icon={SaveIcon} onPress = {() => {
                                 this.setState({edit: false})
                             }}/>
-                            <Text style={styles.rightControlsText}>Save</Text>
+                            <Text style={styles.rightControlsText}>Finish Edit</Text>
```
Comment (coverney) [inline components/home/Home.js:123]: what happened with "exit editing mode"?

**Review by coverney — CHANGES_REQUESTED** [review]
A few minor suggestions

**Comment by coverney** [comment]
One thing is that I am worried that the delete vehicle section button isn't super obvious in terms of what it's purpose is

File: components/home/Welcome.js (hunk 1 above)
Comment (Enmoren) [reply components/home/Welcome.js:29]: Great that can be captured in another card for this sprint!

File: components/home/homeSections/VehicleSection.js (hunk 2 above)
Comment (Enmoren) [reply components/home/homeSections/VehicleSection.js:78]: If we are adding text, I think we should place it in the right corner instead

File: components/home/Home.js (hunk 4 above)
Comment (Enmoren) [reply components/home/Home.js:123]: That sounds like a great option. I would definitely change it to "exit editing mode" if that was an option appearing in our doc.

Comment (coverney) [reply components/home/Home.js:123]: I think it was listed in our Trello card (https://trello.com/c/kKdHKelu)?

Comment (Enmoren) [reply components/home/Home.js:123]: I think I remember that. It just looks way too long

File: components/home/homeSections/VehicleSection.js (hunk 2 above)
Comment (Enmoren) [reply components/home/homeSections/VehicleSection.js:78]: I tried it for a few minutes but was not able to get it to work. I think you are really good at text formatting, we can also move it to the next sprint for sure.

Comment (coverney) [reply components/home/homeSections/VehicleSection.js:78]: Yeah, let's talk about this later, and I can attempt this as one of my cards for next sprint

**Pull request closed without merging**